cvat/apps/engine/views.py

```diff
@@ -533,8 +533,8 @@ def jobs(self, request, pk):
     )
     @action(detail=True, methods=['POST', 'GET'])
     def data(self, request, pk):
+        db_task = self.get_object() # call check_object_permissions as well
         if request.method == 'POST':
-            db_task = self.get_object() # call check_object_permissions as well
```

Comment:
@bsekachev , There is a similar line below. Probably it can be removed as well. I will do that.